### PR TITLE
toaster message added

### PR DESCRIPTION
**Pull request by vinayajoseph-aot**
# Issue Tracking

JIRA: 
Issue Type: BUG/ FEATURE 
https://aottech.atlassian.net/browse/FWF-2107

# Changes
Added success toaster message in draft creation where there is no data change.

# Screenshots (if applicable)
image

### CODE REVIEW

File: forms-flow-web/src/components/Form/Item/View.js [hunk 1]
```diff
@@ -194,6 +197,10 @@ const View = React.memo((props) => {
           })
         );
       }
+      //show success toaster - no datachange, but still draft is createdgit
```
Comment (abhilash-aot) [inline forms-flow-web/src/components/Form/Item/View.js:200]: Can update the Comment to  be more clear.

**Review by abhilash-aot — APPROVED** [review]

File: forms-flow-web/src/components/Form/Item/View.js
Comment (sreehari-aot) [inline forms-flow-web/src/components/Form/Item/View.js:1]: Thought the bug was in draft create method @vinayajoseph-aot

**Review by abilpraju-aot — APPROVED** [review]

File: forms-flow-web/src/components/Form/Item/View.js
Comment (sreehari-aot) [reply forms-flow-web/src/components/Form/Item/View.js:1]: This would trigger the message even if the API is not called right? @vinayajoseph-aot

Comment (vinayajoseph-aot) [reply forms-flow-web/src/components/Form/Item/View.js:1]: yes..For every submit the create api is called, then there is data change is happened then only updatedraft api is called.
So there is a case where no data change occured(submit clicked and without entering any data clicking back). In this case draft is created but no toaster is showing.

**Pull request merged**